util/src/test/java/org/killbill/billing/util/entity/dao/TestDefaultPaginationSqlDaoHelper.java

```diff
@@ -83,6 +83,65 @@ public void testSearch1MatchOnLargeTable() {
         searchAndValidateKombuchas("jasmin", 0L, 100L, 4, 1L, 1L, null, 0L, null);
     }
 
+    @Test(groups = "slow")
+    public void testListKombuchasAll() {
+        insertKombuchas(4);
```

Comment:
The kombuchas are all for the same account today, would be worth inserting rows with other `account_record_id` to validate we don't return some rows for other accounts (and that the count queries are correct).

Reply:
Fixed. The code uses the `accountRecordId` from `internalCallContext` while inserting a `kombucha`. I've added a hack to set the `internalCallContext.accountRecordId` to a dummy value. Let me know if this is okay or if there is a better way to do this.